examples/with-amp/nuxt.config.js

```diff
@@ -8,7 +8,8 @@ const modifyHtml = (html) => {
   html = html.replace('</head>', ampScript + '</head>')
   return html
 }
-export default {
+
+module.exports = {
```

Comment:
Thank you for the fix. 😄 

In `nuxt-edge`(Nuxt.js 2.0 shortly), the `nuxt.config.js` has be refactored into esmodule.

So you can revert this part and upgrade package.json from `nuxt: latest` to `nuxt-edge: latest`

Reply:
I updated dependencies and revert the export part.